docs/components/early-access/alpha/bpmn-copilot/bpmn-copilot.md

```diff
@@ -0,0 +1,31 @@
+---
+id: bpmn-copilot
+title: BPMN copilot
+sidebar_label: Get started
+description: "Using AI, select users can now chat with the BPMN copilot for assistance generating new and functional BPMN process diagrams in Web Modeler based on a process description."
+---
+
+<span class="badge badge--cloud">Camunda 8 SaaS only</span>
+
+:::note Terms of use
+By using this tool, you agree to Camunda's use of the anonymized input and output data and anonymized feedback to improve it.
+:::
+
+Using AI, select users can now chat with the BPMN copilot for assistance generating new and functional BPMN process diagrams in [Web Modeler](/components/modeler/web-modeler/launch-web-modeler.md) based on a process description.
+
+## Get started
+
+:::warning
+Creating a BPMN process diagram with the BPMN copilot will overwrite existing work.
+:::
+
+To use the BPMN copilot in Web Modeler, take the following steps:
+
+1. Log in to [Web Modeler](/components/modeler/web-modeler/launch-web-modeler.md).
+2. Click **New project > Create new > BPMN diagram**. If you have opted in to test this feature, the Camunda copilot chat window will pop up.
+
+![bpmn copilot chat window](./img/bpmn-copilot-chat.png)
+
+3. In the chat box, enter your prompt. This prompt should be a simple, clear, and concise request describing the BPMN diagram you would like to generate. For example, "Generate a mortgage loan process diagram". For more information, you can also ask questions like "How do I design a process?"
```

Comment:
"simple, clear, and concise". 
More complicated requests will take longer to process

Reply:
@christinaausley in this first phase, asking "How do I design a process?" will literally draw a BPMN diagram for you:

![Screenshot 2024-12-13 at 2 37 46 PM](https://github.com/user-attachments/assets/d1f34217-0747-4f01-8235-b0300db2cbb3)

It might be worth removing this last sentence for now to avoid confusion.